### PR TITLE
updated a known issue

### PR DESCRIPTION
**Pull request by anwara-ms**
reported by customers.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/information-barriers-in-teams.md [hunk 1]
```diff
@@ -197,6 +197,8 @@ For more information on licenses and permissions, including plans and pricing, s
...
 
+- **Team owners are not removed**: If a new IB policy is applied, that results in two or more conflicting segments present in a Teams channel, then the segments with Team owners is given higher preferece and other segment users are removed. Also, at this time, Team owners are not getting removed, even if they are in conflict with other owners/users. Tenant admins and other channel owners will have to remove conflicting owners manually. We are working on a fix.
```
Comment (JohanFreelancer9) [inline Teams/information-barriers-in-teams.md:200]:
```suggestion
- **Team owners are not removed**: If a new IB policy is applied that results in two or more conflicting segments present in a Teams channel, the segments with Team owners are given higher preference and other segment users are removed. Also, at this time, Team owners are not getting removed, even if they are in conflict with other owners/users. Tenant admins and other channel owners will have to remove conflicting owners manually. We are working on a fix.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@chrfox PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by chrfox** [comment]
re assigning to @robmazz

**Comment by anwara-ms** [comment]
for your action @robmazz @LauraWi . Please help publish this.

**Comment by robmazz** [comment]
@anwara-ms  - I'll push this live, but I'll remove the "We will fix this" sentence at the end of the addition and republish today.

**Pull request merged**